tests/Api/AssetReferenceFile/DownloadAssetReferenceFileIntegration.php

```diff
@@ -0,0 +1,89 @@
+<?php
+
+namespace Akeneo\PimEnterprise\ApiClient\tests\Api\AssetReferenceFile;
+
+use Akeneo\PimEnterprise\ApiClient\Api\AssetReferenceFileApi;
+use Akeneo\PimEnterprise\ApiClient\tests\Api\ApiTestCaseEnterprise;
+use donatj\MockWebServer\RequestInfo;
+use donatj\MockWebServer\Response;
+use donatj\MockWebServer\ResponseStack;
+use PHPUnit\Framework\Assert;
+use Psr\Http\Message\StreamInterface;
+
+class DownloadAssetReferenceFileIntegration extends ApiTestCaseEnterprise
+{
+    public function test_download_a_localizable_asset_reference_file()
+    {
+        $expectedFilePath = realpath(__DIR__ . '/../../fixtures/ziggy.png');
+
+        $this->server->setResponseOfPath(
+            '/'. sprintf(AssetReferenceFileApi::ASSET_REFERENCE_FILE_DOWNLOAD_URI, 'ziggy', 'en_US'),
+            new ResponseStack(
+                new Response(file_get_contents($expectedFilePath), [], 201)
+            )
+        );
+
+        $api = $this->createClient()->getAssetReferenceFileApi();
+        $api->uploadForLocalizableAsset($expectedFilePath, 'ziggy', 'en_US');
```

Comment:
why do you upload? It is in theory useless

Reply:
Yep... I remove it